packages/manager/modules/log-to-customer/src/live-tail/controller.js

```diff
@@ -0,0 +1,39 @@
+import { LOG_KEYS, DATA_PLATFORM_GUIDE, LOGS_INFO } from '../constants';
+
+export default class LogToCustomerCtrl {
+  /* @ngInject */
+  constructor($translate, LogToCustomerService, coreConfig, coreURLBuilder) {
+    this.$translate = $translate;
+    this.LOG_KEYS = LOG_KEYS;
+    this.LogToCustomer = LogToCustomerService;
+    this.user = coreConfig.getUser();
+    this.coreURLBuilder = coreURLBuilder;
+  }
+
+  $onInit() {
+    this.loading = true;
+
+    this.links = {
+      logPlatformGuide:
+        DATA_PLATFORM_GUIDE[this.user] || DATA_PLATFORM_GUIDE.DEFAULT,
+      logInfo:
+        LOGS_INFO[this.service]?.[this.user.ovhSubsidiary] || LOGS_INFO.DEFAULT,
```

Comment:
Default value should by service or at least with a generic link (current one is dedicated to kubernetes)